Guides/PartialSort.md

```diff
@@ -0,0 +1,52 @@
+# Partial Sort (sortedPrefix)
```

Comment:
I think it would make sense to rename all documents with consistent terminology:

```suggestion
# Sorted Prefix
```